assembl/graphql/schema.py

```diff
@@ -932,7 +932,15 @@ class Query(graphene.ObjectType):
     discussion_preferences = graphene.Field(DiscussionPreferences)
     default_preferences = graphene.Field(DiscussionPreferences)
     locales = graphene.List(Locale, lang=graphene.String(required=True))
+    total_sentiments = graphene.Int()
 
+    def resolve_total_sentiments(self, args, context, info):
+        discussion_id = context.matchdict['discussion_id']
+        discussion = models.Discussion.get(discussion_id)
+        return discussion.db.query(models.SentimentOfPost).filter(
+            models.SentimentOfPost.discussion.has(
+                id=discussion_id)
```

Comment:
You need to add `.filter(models.SentimentOfPost.tombstone_condition())` to not include sentiments from deleted posts I think.